CRR/ReplicationStatusUpdater.js

```diff
@@ -305,7 +306,18 @@ class ReplicationStatusUpdater {
                     this.log.warn(`missing SITE_NAME environment variable, triggering replication to the ${storageClass} storage class`);
                 }
                 return eachLimit(versions, this.workers, (i, apply) => {
-                    const { Key, VersionId } = i;
+                    const { Key, VersionId, IsLatest } = i;
+                    // Skip non-latest versions when CURRENT_VERSION_ONLY is enabled.
```

Comment:
I'm not sure to understand the goal of the comment. The first part for me is not useful as you describe the if condition.
The note I don't understand why it's here as the call to the function is not done here

Reply:
fair enough, i removed the comments